addons/mail/static/tests/tours/mail_activity_schedule_from_chatter.js

```diff
@@ -8,6 +8,25 @@ registry.category("web_tour.tours").add("mail_activity_schedule_from_chatter", {
         {
             trigger: "button:contains('Activities')",
         },
+        {
+            trigger: "input[id='activity_type_id_0']",
+            run: "click",
+        },
+        {
+            trigger: ".dropdown-item:contains('Call')",
+        },
+        {
+            extra_trigger: "button:contains('Schedule')",
+            trigger: "input[id='activity_type_id_0']",
+            run: function (action_helper) {
+                setTimeout(() => {
+                    action_helper.click();
```

Comment:
Nitpicking: isn't there a better way to target the input rather than with `input[id='activity_type_id_0']`?
Notably `id='activity_type_id_0'` can be weak, e.g. in case this is changed and/or some other similar input exist and we have to target `activity_type_id_1` or something.

Probably `id*='activity_type_id'` (`*=` and omitting the `_1`) would be good enough?